### ISSUE TITLE
[4.11.3] Inconsistent behaviour when tap on group avatar in conversation list.

### ISSUE THREAD

**Issue by bme13** [WhisperSystems/Signal-Android#7089]
I have:
- [x] searched open and closed issues for duplicates
- [x] read https://github.com/WhisperSystems/Signal-Android/wiki/Submitting-useful-bug-reports

----------------------------------------

### Bug description
Tapping on group avatars in the conversation list shows inconsistent behavior.  For some groups tapping on the avatar doesn't seem to have any effect, while for some other groups tapping on the group avatar shows details information for a specific contact. This contact is not part of that group.

### Steps to reproduce
- Open signal
- tap on group avatar

**Actual result:** As described for some groups nothing happens, while for others detailed information from a contact, not in the group, is showed. 
**Expected result:** Don't know. Perhaps the behavior that nothing happens is expected, however it should at least not show detailed contact information.

### Device info

**Device:** Samsung SM-G900F
**Android version:** 6.0.1
**Signal version:** 4.11.3

### Link to debug log

In 18:38 tapped on group avatar and info for contact ***99 was shown
In 18:39 tapped on different group avatar and info for contact ***74 was shown

https://gist.github.com/6b925711aa1ede3d076b8a5ad305c0e7

**Comment by bme13** [comment]
Ok, just played a little more and hope this doesn't sound too strange however this weird behavior is what I noticed.
When I open Signal this behavior does not happen. So after opening I see several 1:1 conversations and 3 group avatars in my conversation overview. If I click on a group avatar nothing happens. Now when I scroll down the list until the 13th conversation becomes visible and then scroll up again then the first first group avatar starts the behavior of showing contact info when I tap on it. Now the strange thing is that the contact info it displays is the 12th contact below that group in the conversation list.
As an additional test I've archived one of the contacts between that specific group and the contact and repeated the test. So started signal again, scrolled down to see the 13th conversation visible (which now is a different one) and scrolled back up. Group detail now shows different contact detail which again happens to be the 12th contact info below in the conversation list. 

As described in the first post there were multiple groups that had this behavior. It appears that the other group which shows contact information is further in the list.
To be precise at position 13. And it showed the contact info of the first conversation in the list, which is thus 12 positions up from that group. Now after I've archived the one conversation this specific group doesn't show contact information anymore. Apparently because there is neither a contact above or below at 12 positions. 
Finally I've archived more conversations so that the groups are still there but don't have 12 conversations above or below them in the list. Then I don't have this behavior at all.
It seems that you first need to scroll in the list and then you only see this behavior for group avatars that have a contact in the list with 12 positions offset. Was unable to test what happens if that 12th position was also a group avatar :smile:
